### PR TITLE
Fix dates to have Tooltip with exact date

### PR DESCRIPTION
**Pull request by ZitaNemeckova**
Dates should look like this if possible:

Screenshot 2021-05-03 at 13 02 32

### CODE REVIEW

File: src/components/date-component/date-component.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  isRow?: boolean;
+}
+
+export class DateComponent extends React.Component<IProps> {
+  render() {
+    const { date, isRow } = this.props;
+    return (
+      <Tooltip content={moment(date).format('MMMM Do YYYY')}>
+        {isRow ? (
+          <td>{moment(date).fromNow()}</td>
```
Comment (himdel) [inline src/components/date-component/date-component.tsx:16]: I'm not sure DateComponent should know about tables, would it work to just `<td><DateComponent.../></td>`?

File: src/components/date-component/date-component.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+interface IProps {
+  date: string;
+  isRow?: boolean;
+}
+
+export class DateComponent extends React.Component<IProps> {
+  render() {
+    const { date, isRow } = this.props;
+    return (
+      <Tooltip content={moment(date).format('MMMM Do YYYY')}>
```
Comment (himdel) [inline src/components/date-component/date-component.tsx:14]: Missing a time and timezone? (if the data is available :))

Comment (ZitaNemeckova) [reply src/components/date-component/date-component.tsx:14]: @sbuenafe-rh How should it looklike in tooltip? Thanks :)

**Comment by sbuenafe-rh** [comment]
@ZitaNemeckova the format would be DD MMM YYYY, and if available the timestamp. For example, **07 Jan 2019, 23:33 UTC**

**Review by newswangerd — APPROVED** [review]

**Pull request merged**